Sources/Requests/httpRequest.swift

```diff
@@ -0,0 +1,29 @@
+import Foundation
+
+public class HttpRequest {
+    var method: String
+    var url: String
+    var version: String
+    var headers: [String: String]
+
+    public init(method: String, url: String, version: String, headers: [String: String]) {
+        self.method = method
+        self.url = url
+        self.version = version
+        self.headers = headers
+    }
+
+    public func returnMethod () -> String {
+        return self.method
+    }
+}
+
+extension HttpRequest: Equatable {
```

Comment:
Why do you need this extension?

Reply:
This is how I define what it means for two httpRequest object to be equal